src/modules/UI/scenes/WalletList/components/HwBackButtonHandler/HwBackButtonHandler.android.js

```diff
@@ -0,0 +1,24 @@
+import { ToastAndroid } from 'react-native'
```

Comment:
`Eslint` forces us to separate Android and iOS specific cases to corresponding files. Created same file iOS handler  for consistency, but it will not be used.